.github/workflows/post_peek_screenshot.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs: 
   post_screenshots_in_comment:
     name: Post the screenshot
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
⚠ This change will take effect when this commit is merged to `master`, as for now this action will be loading indefinitely to find a `ubuntu-18.04` machine.